dldashboard/__init__.py

```diff
@@ -58,7 +59,15 @@ def index():
         ip = os.getenv("MCDASHBOARD_IP")
         print("setting MC dashboard ip from env variable MCDASHBOARD_IP={}".format(ip))
 
-    app.run(ip, threaded=True, port=port, debug=False)
+    ssl_cert_file = os.getenv("SSL_CERT_FILE") 
```

Comment:
prefix these with `MCDASHBOARD`, so for example `MCDASHBOARD_SSL_CERT` and ``MCDASHBOARD_SSL_PKEY`